dbos_transact/system_database.py

```diff
@@ -388,16 +396,20 @@ def recv(
                 {
                     "workflow_uuid": workflow_uuid,
                     "function_id": function_id,
-                    "output": rows[0][0],
+                    "output": utils.serialize(
```

Comment:
Didn't we decide not to do that?  Pass in `None` if message is `None`?

Reply:
I realized it won't work if we try to do utils.deserialize(None). Also in this scenario the output exists but is `None`, which is different from the output doesn't exist.